src/app/clark.component.ts

```diff
@@ -65,7 +66,8 @@ export class ClarkComponent implements OnInit {
     private cartService: CartV2Service,
     private router: Router,
     private activatedRoute: ActivatedRoute,
-    private titleService: Title
+    private titleService: Title,
+    private _: HistoryService
```

Comment:
![image](https://user-images.githubusercontent.com/14933100/62631298-7ecbcc00-b8fe-11e9-8e58-8f6357b7acdb.png)

```suggestion
    private _: HistoryService,
```

Reply:
![image](https://user-images.githubusercontent.com/7431390/62632896-3eba1880-b901-11e9-8148-4f5037eea6ce.png)